### ISSUE TITLE
Make some face shields for NHS workers

### ISSUE THREAD

**Issue by amcewen** [DoESLiverpool/somebody-should#1415]
Spinning this out of [Alex's comment](https://github.com/DoESLiverpool/somebody-should/issues/1411#issuecomment-603192225) and [this epic multi-branched Twitter conversation this evening](https://twitter.com/afroleft/status/1242527366804656128), it seems that (at least starting or preparing for) spinning up some manufacturing of face shields is a useful thing we could do to help with the covid-19 pandemic.

## Design considerations
 * [The ones we are using are a trip of foam glued to a clear sheet with elastic to hold it on. They are supposed to be single use.](https://twitter.com/doctorfraz/status/1242558629317926916)
 * [Ideally should fit with other parts of a uniform or alone to protect against splatter/ spray with room for headwear, mask & glasses/goggles. Should be secure around head/adjustable](https://twitter.com/AlyssaAlabassi/status/1242564096454688771)  Here's a [photo of current(?) solutions and the uniform, etc.](https://miro.medium.com/max/1742/1*v6C5oJn1kBSdlUgOFLpQbQ.jpeg)
 * Foam design is [very similar to what we are currently using in ED.
We have brief contact times with patients & get through lots. Other areas e.g. ICU may need something more resilient](https://twitter.com/doctorfraz/status/1242566514454278150)
 * On sterilisation/cleaning: [Should be single use. Might be worth contacting Primary care groups as GPs have no PPE at present so can't see patients many of whom could stay at home](https://twitter.com/doctorfraz/status/1242567550820646914) / [face shields should be able to be decontaminated between patients with bleach- we use acticlor solution in our trust and it's deemed to be good for viral contamination. I think risk of transmission to HCP low and risk of cross contamination 2 next pt lower](https://twitter.com/bom_tarnes/status/1242567517551374342) / [If they're not disposable then they should be easy to clean. I don't know how foam will fare. Depends on where they're being used: everyday outpatients/GP surgery/ pharmacies are different from intensive care units and high infection wards.](https://twitter.com/AlyssaAlabassi/status/1242568887104872448)
 * [To be honest I think that foam head band would be better than the large gap at the top of this one. It is droplet spread so a sneeze or cough while swabbing would go straight through that space 
:sneezing_face:](https://twitter.com/doctorfraz/status/1242576017165815815)

## Materials notes
 * [from my prior involvement in medical 3d printing, PLA and ABS aren't ideal as they end up being porous and trap particulate contaminants. PETG is better and my prior medical designs use this.](https://twitter.com/bom_tarnes/status/1242569448902533122)
 * [Best place for foam sheets is a proper musicians shop. One in Blackpool. Used to line flight cases etc.](https://twitter.com/dunkerley21/status/1242573638454456320)

## Other info, links, etc.
 * [Covid-19 UK collaboration slack](https://covid-19volunteersuk.slack.com/join/shared_invite/zt-czz3p6vp-6zyuIcrbZBIJw1LdiTbffw)

We're also trying to find some links to local primary care groups and hospitals, to work out what's useful for them.

Please update/add to this description as we work things out.

**Comment by amcewen** [comment]
I think @ajlennon and I are going to work on a bit of a prototype to try things out tomorrow.  Remote attendees welcome :-)  Seems like @KarlDunkerley might be poking at it at home too.

**Comment by ajlennon** [comment]
Fantastic curation thanks to @amcewen.

Couple of other bits that came up

I've been speaking with some chaps in Lancaster:

> "I would take direction from any medical folks as they need to take decisions on lowering their safety requirements (i.e. not CE marked).  We've been working locally with folks in Lancaster and Morecambe however you would be under these guys - https://www.liverpoolccg.nhs.uk/ "

So I Googled around and have reached out to

- [Liverpool CCG](https://twitter.com/liverpoolccg) -  [website](https://www.liverpoolccg.nhs.uk/)

- [Liverpool Coronavirus Official](https://twitter.com/COVIDLiverpool) - [website](https://liverpool.gov.uk/communities-and-safety/emergency-planning/coronavirus/)

Not heard back.

I have also heard that Liverpool Uni Additive Labs may be looking at aspects of what can be done to assist the NHS. I have reached out to  [Dr Kate Black](https://twitter.com/DrKateBlack) to ask if the #Maker community in Liverpool can offer any assistance.

A side discussion was that people are saying there *is* PPE equipment but it's in the wrong place. If this is so then why isn't it being moved to the right place and is there a way for us to assist. I will happily rent a van and drive around if this would get PPE to doctors on the front line. #NeedsInvestigation

Lastly if we find we can contribute to rapid manufacture of PPE masks for the local NHS (presumably as part of a wider national/global effort) then where do we source materials and who pays for them? 

I am assuming this is a case of the faster we can manufacture, the more materials and the bigger the crowd the better #scaleability.

**Comment by ajlennon** [comment]
Can do. What time we thinking @amcewen ?

**Comment by amcewen** [comment]
Any time after 12 would be good for me.  Have you got any empty (or emptyable) 2 or 3L coke bottles you could bring as a source of PETG?  I think they'd need to not be ribbed to be easily laserable.  Otherwise we'll need to raid Lidl.

**Comment by ajlennon** [comment]
OK. Post 12. Will buzz you when I'm setting off. Don't drink coke I'm afraid. I wonder who might ? Will think.

**Comment by ajlennon** [comment]
Some more background on needed PPE and much more at the **Internet Book of Critical Care (IBCC)**. Apparently continually being updated.

> gear
> (1) Contact precautions (waterproof gown and gloves)
> (2) Some sort of mask (discussed above in the transmission section)
> N95 mask or a powered, air-purifying respiratory (“PAPR”)
> Surgical mask for patients not undergoing aerosol-generating procedures (based on WHO & Canadian guidelines).
> An anesthesia mask might be placed in-line with a viral filter to MacGyver an N95 mask (see video below).  The main drawback is that the mask itself could become a giant fomite which is contaminated with COVID-19 (so carefully washing the entire mask in between uses).
> (3) Goggles or eye shield.
> (4) Hair cover for aerosol generating procedures (listed above in red text).
> (5) Hood may be used, especially during intubations (this can be constructed out of a trash-bag, as shown in the tweet above).
> Shoes
> Shoe covers aren't recommended, as removing them may increase exposure (ANZICS guidelines).
> Shoes that are easily cleaned and don't need to be touched might be preferable (e.g. Danskos).
> Note:  The exact gear used is probably less important than using it correctly.  

ref: [Internet Book of Critical Care - COVID19](https://emcrit.org/ibcc/COVID19/#personal_protective_equipment_(PPE))

**Comment by KarlDunkerley** [comment]
I am. Does anyone know where to get rolls of double-sided velcro? 
Am thinking of putting slits at the top left and right and threading this through to make shield more adjustable and quicker to assemble. No stapling required.
Thoughts anyone?
May raid local homebase or range if still open.
If anyone comes across any then please let me know.


    On Tuesday, 24 March 2020, 23:19:06 GMT, Adrian McEwen <notifications@github.com> wrote:  
 
 


I think @ajlennon and I are going to work on a bit of a prototype to try things out tomorrow. Remote attendees welcome :-) Seems like @KarlDunkerley might be poking at it at home too.

**Comment by RussCoty** [comment]
Would Velcro be a “germ trap”?

Also just a thought, as a kid my dad used to get us animation cell from work, it might have been Overhead projector film. It was in A4 sheet and sometimes had alignment holes/notches at the top.

**Comment by KarlDunkerley** [comment]
We'd need advice from clinicians but this is the sort of thing I was thinking of...

Velcro® Brand Velcro One-wrap Reusable Ties 30mm x 5m Black | Zoro UK

| 
| 
| 
| £6.79 |  |

 |

 |
| 
|  | 
Velcro® Brand Velcro One-wrap Reusable Ties 30mm x 5m Black | Zoro UK

Velcro® Brand - Velcro One-wrap Reusable Ties 30mm x 5m Black available at www.zoro.co.uk - UK's Leading Supplie...
 |

 |

 |


It has loops on one side and hooks on the other. V easy to thread through a slot on either side.
Foam could also be velcrroed on to mask so that it could be swapped out if it gets dirty.

Random thoughts.
Karl




    On Wednesday, 25 March 2020, 08:27:38 GMT, RussCoty <notifications@github.com> wrote:  
 
 


Would Velcro be a “germ trap”?

Also just a thought, as a kid my dad used to get us animation cell from work, it might have been Overhead projector film. It was in A4 sheet and sometimes had alignment holes/notches at the top.

**Comment by jackie1050** [comment]
Mentioned this simple design to Alex and Adrian - it uses an acetate sheet with a 4 hole punch:

https://twitter.com/3d_maranon/status/1242177410759524355?s=09


I don't think we've got double sided Velcro - if anyone has any use for single sided we've got lots of that (in the fabric cupboard I think).

________________________________

**Comment by RussCoty** [comment]
Laser cutting seems like the answer for high volume, but I feel like it’s best to get a specific agreement with whoever needs the stuff. Could this be solving a problem that doesn’t exist?

**Comment by amcewen** [comment]
@RussCoty good point to raise, but most of the Tweets I linked to in the description are from doctors/NHS staff, so there's at least _some_ demand.  Finding people to talk to locally to work out demand (or if there's more important stuff to be tackling) is key though.

Hopefully by the time we get to have such a conversation we'll at least have a better idea of what we can offer though, and maybe a prototype to show them.

**Comment by KarlDunkerley** [comment]
Is it the stick on or sew-on type velcro and in a roll or a sheet?
    On Wednesday, 25 March 2020, 08:41:31 GMT, Jackie Pease <notifications@github.com> wrote:  
 
 

Mentioned this simple design to Alex and Adrian - it uses an acetate sheet with a 4 hole punch:

https://twitter.com/3d_maranon/status/1242177410759524355?s=09


I don't think we've got double sided Velcro - if anyone has any use for single sided we've got lots of that (in the fabric cupboard I think).

________________________________

**Comment by jackie1050** [comment]
Not sure - maybe some of both
________________________________

**Comment by KarlDunkerley** [comment]
You've given me a tought. I have some printable clear transparency sheets for inkjets. They are flexible but not ridiculously so.
Only slight issue is that the printable surface is very slightly hazy.
This sort of thing...
OfficeTree Overhead Projector Film - 50 Acetate Sheets A4 -Transparent - Laser - 692624403116 | eBay

| 
| 
| 
| £20.34 |  |

 |

 |
| 
|  | 
OfficeTree Overhead Projector Film - 50 Acetate Sheets A4 -Transparent -...

Click here to see description.
 |

 |

 |


But mine is A3.




    On Wednesday, 25 March 2020, 08:27:38 GMT, RussCoty <notifications@github.com> wrote:  
 
 


Would Velcro be a “germ trap”?

Also just a thought, as a kid my dad used to get us animation cell from work, it might have been Overhead projector film. It was in A4 sheet and sometimes had alignment holes/notches at the top.

**Comment by KarlDunkerley** [comment]
My mate in packaging has suggested this firm for PETG. Have emailed them requesting help.
PET | Plastic Products Limited

| 
| 
| 
|  |  |

 |

 |
| 
|  | 
PET | Plastic Products Limited

PPL offers an extensive range of BOPET films for a wide range of applications in addition to APET for box making...
 |

 |

 |



Cheers,




    On Wednesday, 25 March 2020, 10:00:09 GMT, Jackie Pease <notifications@github.com> wrote:  
 
 

Not sure - maybe some of both
________________________________

**Comment by ajlennon** [comment]
This looks to be the kind of thing

https://twitter.com/pieteriow/status/1242444947208486912?s=21

**Comment by KarlDunkerley** [comment]
Managed to pick up two rolls of this plus some sticky back velcro from Homebase. 20 min wait to get into store.
Will have a go tonight after work.
    On Wednesday, 25 March 2020, 11:27:45 GMT, Alex Lennon <notifications@github.com> wrote:  
 
 


This looks to be the kind of thing

https://twitter.com/pieteriow/status/1242444947208486912?s=21

**Comment by amcewen** [comment]
Running a test print of a [3D printed design in use in Spain](https://twitter.com/3d_maranon/status/1242177410759524355) that @jackie1050 found, alongside the other experiments this afternoon.

![image](https://user-images.githubusercontent.com/108468/77546613-083b0900-6ea4-11ea-84c8-342507481755.png)

**Comment by KarlDunkerley** [comment]
Anyone got any samples of the flexible clear plastic that's laserable.

    On Wednesday, 25 March 2020, 14:23:31 GMT, Adrian McEwen <notifications@github.com> wrote:  
 
 


Running a test print of a 3D printed design in use in Spain that @jackie1050 found, alongside the other experiments this afternoon.

**Comment by ajlennon** [comment]
Question from Liam at Astral

> guy at hospital here is very keen to get a price for face shield asap. he assures me hospital management have no better idea that he does about where theyre gonna get them from.

**Comment by amcewen** [comment]
I've got one sheet of mylar (mostly clear), could do with finding some more and/or a source of clear PET or PETG sheet.  It'd be good to work out how thick/flexible the proper face shields' clear visors are.

Current stash of stuff to try using - https://twitter.com/amcewen/status/1242826238630248449

**Comment by KarlDunkerley** [comment]
Contacted one firm who do 2mm PETG but they reckon it is too stiff to bend.
I'm working on less than 1mm. My workshop face shield is 0.7mm according to micrometer.





    On Wednesday, 25 March 2020, 15:03:00 GMT, Adrian McEwen <notifications@github.com> wrote:  
 
 


I've got one sheet of mylar (mostly clear), could do with finding some more and/or a source of clear PET or PETG sheet. It'd be good to work out how thick/flexible the proper face shields' clear visors are.

Current stash of stuff to try using - https://twitter.com/amcewen/status/1242826238630248449

**Comment by KarlDunkerley** [comment]
Someone on a laser forum replied to my post and suggested this company in leeds.
http://ppb.uk.com/Catalogue/petg-apet-antiglare-apet/petg-gloss-clear-pe-filmed-both-sides/700-micron-clear-petg

Have emailed them and awaiting response.
Cheers.
    On Wednesday, 25 March 2020, 15:03:00 GMT, Adrian McEwen <notifications@github.com> wrote:  
 
 


I've got one sheet of mylar (mostly clear), could do with finding some more and/or a source of clear PET or PETG sheet. It'd be good to work out how thick/flexible the proper face shields' clear visors are.

Current stash of stuff to try using - https://twitter.com/amcewen/status/1242826238630248449

**Comment by KarlDunkerley** [comment]
Has anyone seen this info?
https://pwp.gatech.edu/rapid-response/face-shields/



Very interesting.
    On Wednesday, 25 March 2020, 15:03:00 GMT, Adrian McEwen <notifications@github.com> wrote:  
 
 


I've got one sheet of mylar (mostly clear), could do with finding some more and/or a source of clear PET or PETG sheet. It'd be good to work out how thick/flexible the proper face shields' clear visors are.

Current stash of stuff to try using - https://twitter.com/amcewen/status/1242826238630248449

**Comment by ajlennon** [comment]
>Has anyone seen this info?

Can be laser cut. Excellent. Needs investigation

**Comment by KarlDunkerley** [comment]
https://l.facebook.com/l.php?u=https%3A%2F%2Fwww.prusaprinters.org%2Fprints%2F25857-prusa-protective-face-shield-rc2%2Ffiles%3Ffbclid%3DIwAR1N3Bopha12MQg79AK7p93T9jcmxr80VC5J_jmrJPooTfpB5-Z8IWLEMv0&h=AT1U0o9O1_FSf2jApc9-wlIoOmc_vUh-SzuJfI2tao3xL1FlmJnq9xdMYMC94tCGEpKnOKWpzmEAPqe143CSOU9To-lx0ukFGiJ2DPPgxQuuX25697ta-34bsJiwL2qzc9eH5g

More ideas...
    On Wednesday, 25 March 2020, 17:45:06 GMT, Alex Lennon <notifications@github.com> wrote:  
 
 



Has anyone seen this info?


Can be laser cut. Excellent. Needs investigation

**Comment by KarlDunkerley** [comment]
First prototype....It smells a bit!

About 10 mins max to cut and make but that was with OHP film not PETG. Can get two visors on A3 sheet.
The foam is from Halfords floor tiles and the velcro was a mix of sticky (for the replaceable forehead band) and the Reusable Tie Velcro for the adjustable straps.


Thoughts everyone?






    On Wednesday, 25 March 2020, 17:45:06 GMT, Alex Lennon <notifications@github.com> wrote:  
 
 



Has anyone seen this info?


Can be laser cut. Excellent. Needs investigation

**Comment by KarlDunkerley** [comment]
Instructions;

Items Needed(For prototype)
Sheets of OHP INKjet Printable film (Prototype only) for visor [PETG possibly for prodn version]EG https://www.ebay.co.uk/itm/100-x-OHP-sheets-for-Overhead-Projector-100-micron-Transparent-Printable-A3-A4/123949546417?var=424740269322&hash=item1cdbf7fbb1:m:mRHTIRuGJND2oGowlTrf9xA
EVA Foam for forehead bandeg https://www.halfords.com/tools/garage-equipment/garage-essentials/rolson-6-piece-floor-mat-set---120-x-180cm-552101.html
Reusable velcro for adjustable side strapsEg VELCRO® Brand ONE-WRAP® Reusable Ties 30mm x 5m - Blackhttps://www.homebase.co.uk/velcro-brand-one-wrap-reusable-ties-30mm-x-5m-black_p210085
Stick on Velcro to attach Forehead foamEg VELCRO Stick On Tape 20mm x 2.5mhttps://www.homebase.co.uk/velcro-stick-on-tape-20mm-x-2-5m-_p510701

Step 1 
Download files Visor.dxf and Forehead.dxf (or Lightburn equivalents)

Step 2
Experiment with your laser settings to cut visor. 
Designed to fit one on A4 or two on A3. 
Dimensions are 290mm x 205mm

Step 3
Experiment with your laser settings to cut foam cleanly.
Dimensions are 25mm x 195mm
Can cut lots from single sheet of foam.


Step 4
Cut VELCRO Stick On Tape to just less than foam.
Pull of paper and attach both together to Foam.
Carefully place faom equidistant to the two strap slits.

Step 5
Cut a piece of the Reusable Velcro about 26cm long (or more for big heads LOL)
Feed Reusable Velcro through one slit with loop/furry side towards visor (smoother on bonce)
You need to double the strap back on itself so that it sticks to itself.
This holds the strap in place. You may have to experiment as difficult to explain.

Step 6
Do the same on the other side.

Step 7
Et Voila.





    On Wednesday, 25 March 2020, 17:45:06 GMT, Alex Lennon <notifications@github.com> wrote:  
 
 



Has anyone seen this info?


Can be laser cut. Excellent. Needs investigation

**Comment by markmellors** [comment]
Hi, I was pointed here by Alex, on Twitter. I've been trying to get my head around visor requirements over the last week. Here's what I've picked up so far:
I think this is the best description of the requirements: https://docs.google.com/document/d/15k20PMe14_XQa-8kTA2G7nSIrF1bGSpc90_GM9H8yNI/edit
I know that's US centric, but it matches up well with the product that a UK hospital (Addenbrookes) told me they were using.
I think elastic is prefered over velcro, for one handed operation
I think a foam pad is preferred for comfort, over a 3d print, and it also serves as a brow sheild.
I agree with what you've said above about sterilisation and disposablity. I'm still waiting to hear back about how possible sterilisation is and why they don't currently do it. 
I've bought some 0.5mm PET sheet (8x4'!) to start with, about £10/sheet, bayplastics.co.uk, so not really worth faffing with pop bottles. I think thinner is better/cheaper, if it can be sourced. 0.18mm is what's used in the delve design. I haven't found a low volume source yet. 

My current estimate on numbers is ~5/day per staff member and 1 staff per 2-4 patients (at hospitals, I have no idea about GPs or paramedics). so volumes will soon exceed what hobbyists can supply, but hopefully we can plug the gap before a higher volume supply can be established.

**Comment by ajlennon** [comment]
That is really really helpful information - thanks @markmellors. 

We're just working to put out a 3rd video which will be the last tonight then I think @amcewen has plans to edit this thread. It'll be interesting to see how the design can sit with the requirements you list above.

**Comment by KarlDunkerley** [comment]
Mark, 
Do you have any Pet spare that I could purchase off you?
Just want to test prototype and cutting speeds.
Cheers,
    On Wednesday, 25 March 2020, 21:26:56 GMT, markmellors <notifications@github.com> wrote:  
 
 


Hi, I was pointed here by Alex, on Twitter. I've been trying to get my head around visor requirements over the last week. Here's what I've picked up so far:
I think this is the best description of the requirements: https://docs.google.com/document/d/15k20PMe14_XQa-8kTA2G7nSIrF1bGSpc90_GM9H8yNI/edit
I know that's US centric, but it matches up well with the product that a UK hospital (Addenbrookes) told me they were using.
I think elastic is prefered over velcro, for one handed operation
I think a foam pad is preferred for comfort, over a 3d print, and it also serves as a brow sheild.
I agree with what you've said above about sterilisation and disposablity. I'm still waiting to hear back about how possible sterilisation is and why they don't currently do it.
I've bought some 0.5mm PET sheet (8x4'!) to start with, about £10/sheet, bayplastics.co.uk, so not really worth faffing with pop bottles. I think thinner is better/cheaper, if it can be sourced. 0.18mm is what's used in the delve design. I haven't found a low volume source yet.

My current estimate on numbers is ~5/day per staff member and 1 staff per 2-4 patients (at hospitals, I have no idea about GPs or paramedics). so volumes will soon exceed what hobbyists can supply, but hopefully we can plug the gap before a higher volume supply can be established.

**Comment by markmellors** [comment]
I'm still waiting for my PET to arrive. Was meant to be here today. I should be able to send you enough for one visor though. Straight to DoES?

**Comment by KarlDunkerley** [comment]
Whereabouts are you? I'm in Southport/Ainsdale. Can drive over maybe.

    On Wednesday, 25 March 2020, 22:13:57 GMT, markmellors <notifications@github.com> wrote:  
 
 


I'm still waiting for my PET to arrive. Was meant to be here today. I should be able to send you enough for one visor though. Straight to DoES?

**Comment by markmellors** [comment]
lol. Hitchin, Hertfordshire

**Comment by KarlDunkerley** [comment]
Bugger!
My Machine can do 500mm x 300mm so any sheets around that or A3 appreciated. Can get two visors from an A3 with my design.
Cheers
    On Wednesday, 25 March 2020, 22:17:40 GMT, markmellors <notifications@github.com> wrote:  
 
 


lol. Hitchin, Hertfordshire

**Comment by KarlDunkerley** [comment]
Have contacted this chap about 0.18mm PET.
APET and RPET reels - Stephen Webster Plastics

| 
| 
| 
|  |  |

 |

 |
| 
|  | 
APET and RPET reels - Stephen Webster Plastics

Virgin APET or Recycled RPET reels for packaging in 0.18mm to 1.1mm and almost any width. Call our sales team fo...
 |

 |

 |








    On Wednesday, 25 March 2020, 21:26:56 GMT, markmellors <notifications@github.com> wrote:  
 
 


Hi, I was pointed here by Alex, on Twitter. I've been trying to get my head around visor requirements over the last week. Here's what I've picked up so far:
I think this is the best description of the requirements: https://docs.google.com/document/d/15k20PMe14_XQa-8kTA2G7nSIrF1bGSpc90_GM9H8yNI/edit
I know that's US centric, but it matches up well with the product that a UK hospital (Addenbrookes) told me they were using.
I think elastic is prefered over velcro, for one handed operation
I think a foam pad is preferred for comfort, over a 3d print, and it also serves as a brow sheild.
I agree with what you've said above about sterilisation and disposablity. I'm still waiting to hear back about how possible sterilisation is and why they don't currently do it.
I've bought some 0.5mm PET sheet (8x4'!) to start with, about £10/sheet, bayplastics.co.uk, so not really worth faffing with pop bottles. I think thinner is better/cheaper, if it can be sourced. 0.18mm is what's used in the delve design. I haven't found a low volume source yet.

My current estimate on numbers is ~5/day per staff member and 1 staff per 2-4 patients (at hospitals, I have no idea about GPs or paramedics). so volumes will soon exceed what hobbyists can supply, but hopefully we can plug the gap before a higher volume supply can be established.

**Comment by ajlennon** [comment]
Videos of the work today

These are all first passes, all have pros and cons, but I confess I do like what Adrian has come up with here in #3

(1) NHS PPE eye protector evaluation #1 - 3D Printed Visor

https://youtu.be/WjN5fSOK3HE

(2) NHS PPE eye protector evaluation #2 - Laser Cut Visor

https://youtu.be/i4zo93ani40

(3) NHS PPE eye protector evaluation #3 - Optimised Laser Cut Visor

https://youtu.be/2b9gzPKdUrw

**Comment by KarlDunkerley** [comment]
BUGGER!!! Hoped this org had some 0.18mm PETG we could use but they are already using it.
See below.....


Hi,
Any idea who might have some available?


Cheers,

Karl Dunkerley

Mob 07710 353 122

eml karldunkerley@yahoo.com

On Thursday, 26 March 2020, 07:58:29 GMT, Lauson Higgins <lauson@stephen-webster.co.uk> wrote:

Hello Karl

Thank you for your email this morning

We are also working as part of a collective in production of visors specifically destined for NHS direct / non profit

All going to plan we’ll mass produce 80,000 Face Sheilds and I know you will appreciate how that takes priority on material allocation ect

As you can imagine, all clear substrates are taking a bit of a kicking on availability

Really appreciate your communication

Keep doing what you are doing !


Lauson
BDM Stephen Webster Plastics
Email:      Lauson@stephen-Webster.com
Mobile:    +447415241115
Office:      01727 863 138



    On Wednesday, 25 March 2020, 22:45:09 GMT, Alex Lennon <notifications@github.com> wrote:  
 
 


Videos of the work today

These are all first passes, all have pros and cons, but I confess I do like what Adrian has come up with here in #3

(1) NHS PPE eye protector evaluation #1 - 3D Printed Visor

https://youtu.be/WjN5fSOK3HE

(2) NHS PPE eye protector evaluation #2 - Laser Cut Visor

https://youtu.be/i4zo93ani40

(3) NHS PPE eye protector evaluation #3 - Optimised Laser Cut Visor

https://youtu.be/2b9gzPKdUrw

**Comment by RussCoty** [comment]
Did you all see the Trotec email the other day with the lanyard door handle grabber? a simple laser cut design. 

Might be worth leveraging Trotec's mailing list to get this out to laser cutter operators worldwide.i think this might be of extreme use in less wealthy countries where mediacal staff are even more streched for PPE

**Comment by KarlDunkerley** [comment]
WOOHHHOOOOO!!!

Sort of...

Have found a supplier of 400 micron PETGAG in 1040 x 1550 in stock at £2.56 per sheet plus VAT. With my design that is 25 to a sheet and 10p each.
400microns compares to 180 microns for OHP transparency so a good thickness to use and laser.
However, for health and safety, they will only ship in pallet loads of around 870 sheets. That's around 22,000 visors we could make from it.
This means an outlay of approx £2,600 per pallet. 
I can probably get hold of a van and pallet shifter. 
Thoughts anyone?

KARL



    On Wednesday, 25 March 2020, 22:45:09 GMT, Alex Lennon <notifications@github.com> wrote:  
 
 


Videos of the work today

These are all first passes, all have pros and cons, but I confess I do like what Adrian has come up with here in #3

(1) NHS PPE eye protector evaluation #1 - 3D Printed Visor

https://youtu.be/WjN5fSOK3HE

(2) NHS PPE eye protector evaluation #2 - Laser Cut Visor

https://youtu.be/i4zo93ani40

(3) NHS PPE eye protector evaluation #3 - Optimised Laser Cut Visor

https://youtu.be/2b9gzPKdUrw

**Comment by RussCoty** [comment]
I would assume that final assembly would be done on site at the medical facility? 

Just another crazy thought....is there a way that the sheet can be cut to be the whole mask? I can't envision it myself but I thought it was worth putting out there. A single sheet die cut that kind of unwraps to connect around the head....remember those old card caps that you used to get in the 80s? That kind of thing.

**Comment by RussCoty** [comment]
Also @amcewen you had a 2 hole punch but I guess those units are made for a 4 hole punch? 

Could your laser cut band have notches to accomodate holes instead? like little upside-down U shapes:

.....................................................
      n          n            n           n  
.....................................................
|                                                    |
|                                                    |
|                                                    |
|                                                    |
|                                                    |
|                                                    |
|                                                    |
|                                                    |
|                                                    |
___________________________________

**Comment by KarlDunkerley** [comment]
Mine could go out as components to be quickly assembled but can be done in a minute or so.
Components would ship in smaller boxes as the forehead standoff foam would take space up.
Foams could go separate and the rest assembled I guess but v simple to do either way.


    On Thursday, 26 March 2020, 09:04:43 GMT, RussCoty <notifications@github.com> wrote:  
 
 


I would assume that final assembly would be done on site at the medical facility?

Just another crazy thought....is there a way that the sheet can be cut to be the whole mask? I can't envision it myself but I thought it was worth putting out there. A single sheet die cut that kind of unwraps to connect around the head....remember those old card caps that you used to get in the 80s? That kind of thing.

**Comment by ajlennon** [comment]
Great work @KarlDunkerley !

**Comment by RussCoty** [comment]
I guess the next step is to confirm that the services actually will use them?

**Comment by ajlennon** [comment]
Absolutely @RussCoty - hoping to get somebody on a call today. Fingers crossed

**Comment by RussCoty** [comment]
Great work @ajlennon, wondering how to best get this info out to other countries too.

**Comment by ajlennon** [comment]
Absolutely @markmellors has been asking the same kinds of questions

**Comment by RussCoty** [comment]
Press releases might be the answer

**Comment by KarlDunkerley** [comment]
Bugger!
Chap just called to say he's sorry but all that 400micron has been hoovered up by a company in the midlands and only 1 pallet of 220micron left.
Sending price.
Should be thick enough but I suspect events may overtake us all.

Cheers,


    On Thursday, 26 March 2020, 09:29:20 GMT, RussCoty <notifications@github.com> wrote:  
 
 


Press releases might be the answer

**Comment by KarlDunkerley** [comment]
Chap has said these are the 220micron prices;

Sorry to break the bad news.
The up side is we have 220mic PETGAG in 1040 x 1550 at £143.67 per 100 sheets plus VAT, 3672 sheets in stock. 1170 sheets per pallet.
My visors would cost 0.5p each I think. Maths maybe wonky. 25 to a sheet. 
Could someone check the maths for me?
Cheers

    On Thursday, 26 March 2020, 09:29:20 GMT, RussCoty <notifications@github.com> wrote:  
 
 


Press releases might be the answer

**Comment by KarlDunkerley** [comment]
Sooner the better if we want to get any supplies.

See other emails.


    On Thursday, 26 March 2020, 09:16:19 GMT, Alex Lennon <notifications@github.com> wrote:  
 
 


Absolutely @RussCoty - hoping to get somebody on a call today. Fingers crossed

**Comment by KarlDunkerley** [comment]
Can you ask about Polypropylene?
   
Might be able to source rolls of that.
Cheers,


 On Thursday, 26 March 2020, 09:16:19 GMT, Alex Lennon <notifications@github.com> wrote:  
 
 


Absolutely @RussCoty - hoping to get somebody on a call today. Fingers crossed

**Comment by KarlDunkerley** [comment]
From: George I Fomitchev <reangeorge.mail.ru@mail.mailchimpapp.com>To: "karldunkerley@yahoo.com" <karldunkerley@yahoo.com>Sent: Thursday, 26 March 2020, 10:26:20 GMTSubject: After yesterdays newsletter. A new open-source laser air cleaner from Endurance. Kills bacterias and viruses.
Not directly relevant but may be of interest.



| 
| 
| 
| 
|  |

 |

 |

 |
| 
| 
| 
| 
We are making a laser air cleaning device to protect indoors from bacterias and viruses! Fully open-source!
 |

 |


| 
|  |

 |


| 
| 
| 
| 
| 
| 
|  | Share |

 |

 |


| 
| 
|  | Tweet |

 |

 |


| 
| 
|  | Forward |

 |

 |


| 
| 
|  | +1 |

 |

 |


| 
| 
|  | Read Later |

 |

 |


| 
| 
|  | Pin |

 |

 |


| 
| 
|  | Share |

 |

 |

 |

 |

 |

 |


| 
| Learn more about it |

 |

 |

 |

 |



    On Thursday, 26 March 2020, 09:29:20 GMT, RussCoty <notifications@github.com> wrote:  
 
 


Press releases might be the answer

**Comment by ajlennon** [comment]
This is very interesting. 

https://www.yumpu.com/en/document/read/63174168/industry-call-out-250320

**Comment by KarlDunkerley** [comment]
PREMIUM WHITE/BLACK Flat WOVEN ELASTIC BAND SEWING GARMENT VARIOUS LENGTHS/WIDTH | eBay

| 
| 
| 
| £0.99 |  |

 |

 |
| 
|  | 
PREMIUM WHITE/BLACK Flat WOVEN ELASTIC BAND SEWING GARMENT VARIOUS LENGT...

OUR PREMIUM QUALITY FLAT WOVEN ELASTIC BANDS ARE. Our manufacturers make sure The Stretch test and when sewn in ...
 |

 |

 |


30m of 20mm Black £12.99




    On Thursday, 26 March 2020, 15:37:27 GMT, Alex Lennon <notifications@github.com> wrote:  
 
 


This is very interesting.

https://www.yumpu.com/en/document/read/63174168/industry-call-out-250320

**Comment by jackie1050** [comment]
Have people seen this Slack group? Looks like they've got visor material to share:



[Image]

________________________________

**Comment by KarlDunkerley** [comment]
Have just ordered rather a lot of elastic and clear polypropylene if anyone needs some for prototyping.
Should be here in next few days.
Free of charge of course.
Let me know.
    On Thursday, 26 March 2020, 15:59:18 GMT, Jackie Pease <notifications@github.com> wrote:  
 
 

Have people seen this Slack group? Looks like they've got visor material to share:



[Image]

________________________________

**Comment by ajlennon** [comment]
Feedback from my CE guy

>Quick chat with PPE expert
Apparently medical face shields don’t provide sufficient coverage ( up / down / wrap around to temples) as recommended by WHO
Applicable standard is EN 166
PPE needs 3rd party testing and NB certificate 
Approximately £2-2.5 k
Standard from Estonian Standards https://www.evs.ee/en/search

**Comment by ajlennon** [comment]
This is the applicable CE standard (free to download)

https://www.evs.ee/en/evs-en-166-2003

**Comment by ajlennon** [comment]
Just had another very interesting conversation on certification.

Key points are:

- What's the material around headband. We need bio-compatibility so if we are using something that is in regular use we can assume biocompatibility.

- We need to be clear about what the device does and does not do.

- A medical device protects the PATIENT. We are not making a device to protect the patient. 

- A PPE device protects the wearer against impact and/or projectiles. We are not protecting the wearer against impact. 

** Given the above points we do not need CE PPE certification **

- This is a  plash visor to provide partial protection to doctor not patient. Not a medical device

- We can then place it on the market having done due consideration as to what is applicable. (That said it may help to look at impact strength of material and document)

- There should be data sheet that doesn't make any claims that can't be substantiated. Just spray impact.

- It will help to have the materials spec. for what we are cutting i.e. material supply. Put this in data sheet

**Comment by KarlDunkerley** [comment]
Excellent stuff.
Do get the impression that not all clinicians and authorities saying the same thing or it might be they are talking at cross purposes.
What you've written clarifies a lot of stuff for me.
Have emailed those Welsh people to see if my design is suitable (with mods if necessary).
Waiting to hear.
Cheers,
    On Thursday, 26 March 2020, 17:13:17 GMT, Alex Lennon <notifications@github.com> wrote:  
 
 


Just had another very interesting conversation on certification.

Key points are:
   
   -    
What's the material around headband. We need bio-compatibility so if we are using something that is in regular use we can assume biocompatibility.

   -    
We need to be clear about what the device does and does not do.

   -    
A medical device protects the PATIENT. We are not making a device to protect the patient.

   -    
A PPE device protects the wearer against impact and/or projectiles. We are not protecting the wearer against impact.


** Given the above points we do not need CE PPE certification **
   
   -    
This is a plash visor to provide partial protection to doctor not patient. Not a medical device

   -    
We can then place it on the market having done due consideration as to what is applicable. (That said it may help to look at impact strength of material and document)

   -    
There should be data sheet that doesn't make any claims that can't be substantiated. Just spray impact.

   -    
It will help to have the materials spec. for what we are cutting i.e. material supply. Put this in data sheet

**Comment by ajlennon** [comment]
>Excellent stuff.
Do get the impression that not all clinicians and authorities saying the same thing or it might be they are talking at cross purposes.
What you've written clarifies a lot of stuff for me.

Yes - I'm working on the basis some trusts will say "great give us them". Others will say "No we need XYZ". So I am prepping the argument to make to cover what is required in the hope it makes it easy for trusts then to make use of what we can collectively provide

**Comment by KarlDunkerley** [comment]
Spoke to two nurse friend. One a theatre airways person for whom nowhere near enough.
Other just qualified and she has nothing at the moment. Loved the design and desperate for anything.
BTW Post Offices now starting on reduced hours. Not directly related but worth noting.


    On Thursday, 26 March 2020, 17:19:25 GMT, Alex Lennon <notifications@github.com> wrote:  
 
 



Excellent stuff.
Do get the impression that not all clinicians and authorities saying the same thing or it might be they are talking at cross purposes.
What you've written clarifies a lot of stuff for me.


Yes - I'm working on the basis some trusts will say "great give us them". Others will say "No we need XYZ". So I am prepping the argument to make to cover what is required in the hope it makes it easy for trusts then to make use of what we can collectively provide

**Comment by KarlDunkerley** [comment]
Serious question.
Contact saying he may be able to get his Trust to fund materials.
Would we really be able to churn them out?
Mike G reckons Gerald can be kept operational.
Shame Monty not operational.
My 50W CO2 tube still available if needed.
Thoughts?
karl

    On Thursday, 26 March 2020, 17:19:25 GMT, Alex Lennon <notifications@github.com> wrote:  
 
 



Excellent stuff.
Do get the impression that not all clinicians and authorities saying the same thing or it might be they are talking at cross purposes.
What you've written clarifies a lot of stuff for me.


Yes - I'm working on the basis some trusts will say "great give us them". Others will say "No we need XYZ". So I am prepping the argument to make to cover what is required in the hope it makes it easy for trusts then to make use of what we can collectively provide

**Comment by amcewen** [comment]
So much discussion and activity!  :tada: :heart_eyes: 

I don't know about anyone else, but I'm getting a bit overwhelmed with trying to keep up with it all in a single thread.  So.  I've created a new repository to (a) hold information about things we've found out and links to other useful stuff, and (b) have its own [issues area](https://github.com/DoESLiverpool/covid19/issues) where we can separate the things-to-do out into individual issues.

So, please check out [DoESLiverpool/covid19](https://github.com/DoESLiverpool/covid19) and hit the "Watch" button at the top of it if you want to get emails with any activity.

And please help with migrating and sorting the information we've got in this issue over to the new repository! :pray: :+1: :fist_raised: 

(it's also going to give me somewhere to host the design files for my polypropylene design :grin:)

**Comment by ajlennon** [comment]
>Serious question.
Contact saying he may be able to get his Trust to fund materials.
Would we really be able to churn them out?

I absolutely believe we can make a significant distance. Taking money has to be handled with care but I have been approached about donations and we are definitely going to need money for consumables.

**Comment by KarlDunkerley** [comment]
I'm talking to someone who may be able to get the materials funded. Directlt would be best and then delivered to us.
Cut out the middle man.
Did online meeting happen?

    On Thursday, 26 March 2020, 20:28:07 GMT, Alex Lennon <notifications@github.com> wrote:  
 
 



Serious question.
Contact saying he may be able to get his Trust to fund materials.
Would we really be able to churn them out?


I absolutely believe we can make a significant difference. Taking money has to be handled with care but I have been approached about donations and we are definitely going to need money for consumables.

**Comment by ajlennon** [comment]
@KarlDunkerley happening now!

Live chat now here: https://meet.jit.si/bubblino

I setup a group here to manage comms. Would be grateful if you'd join us:

https://groups.google.com/d/forum/does-covid-response

**Comment by magman2112** [comment]
I’ve been a bit distant from these conversations recently, but I will jump in here for some of the more practical aspects.

If we work on a laser cut version, then I can volunteer for possible shifts in laser cutting as well as any usual laser maintenance.

I also have a car, so can look at deliveries, material distribution, etc., regardless of the production models that are chosen.

**Comment by KarlDunkerley** [comment]
Didn't work on my pC. Only realised later it would work on my phone.
Having issues with other vidcon platforms too.

    On Thursday, 26 March 2020, 20:57:54 GMT, Alex Lennon <notifications@github.com> wrote:  
 
 


@KarlDunkerley happening now!

Live chat now here: https://meet.jit.si/bubblino

I setup a group here to manage comms. Would be grateful if you'd join us:

https://groups.google.com/d/forum/does-covid-response

**Comment by ajlennon** [comment]
I think we're also now revisiting a no-print foam idea as a basis for a splash protector mask. This could minimise machine time potentially and be more of an assembly production line task

**Comment by KarlDunkerley** [comment]
I've also been thinking about the 3D priniting people out there. I'm not one of them but I was concerned at the print times so I've come up with a hybrid suggestion.
If you look at the pic of my original visor below, you will see the slits for the velcro or elastic to go through.
Foam will be attached between the slits to form the brow bar.
But what if the slits were used to attach something like the wings/arms of a pair of spectacles that go over ears?
See the Side/arm and end tips shown below in second pic. This is the bits I'm clumsily trying to refer to.

This would use much less filament, be much faster and you could get more on the printing platen.
Might need small medium and large but what do the 3D printing people think? 
Could even be a punched hole and stud/rivet design.
Any thoughts?





See the Side/arm and end tips shown below. This is the bits I'm clumsily trying to refer to.




    On Thursday, 26 March 2020, 23:56:48 GMT, Alex Lennon <notifications@github.com> wrote:  
 
 


I think we're also now revisiting a no-print foam idea as a basis for a splash protector mask. This could minimise machine time potentially and be more of an assembly production line task

**Comment by KarlDunkerley** [comment]
DIY FACE SHIELD

| 
| 
| 
|  |  |

 |

 |
| 
|  | 
DIY FACE SHIELD

if you don't like this Video Just ignore it. No hate just Love. Loving is Caring. Let's Fight NCovid19. Material...
 |

 |

 |






    On Thursday, 26 March 2020, 23:56:48 GMT, Alex Lennon <notifications@github.com> wrote:  
 
 


I think we're also now revisiting a no-print foam idea as a basis for a splash protector mask. This could minimise machine time potentially and be more of an assembly production line task

**Comment by KarlDunkerley** [comment]
Endurance builds a sealed chamber. An Airtight camera (assembling process) – EnduranceLasers

| 
| 
| 
|  |  |

 |

 |
| 
|  | 
Endurance builds a sealed chamber. An Airtight camera (assembling proces...


 |

 |

 |






    On Thursday, 26 March 2020, 23:56:48 GMT, Alex Lennon <notifications@github.com> wrote:  
 
 


I think we're also now revisiting a no-print foam idea as a basis for a splash protector mask. This could minimise machine time potentially and be more of an assembly production line task

**Comment by KarlDunkerley** [comment]
Has everyone sen this?
Medics protective gear guidance 'to be updated'

| 
| 
| 
|  |  |

 |

 |
| 
|  | 
Medics protective gear guidance 'to be updated'

New rules on when health staff should wear masks and gloves are expected amid the Covid-19 crisis.
 |

 |

 |






    On Thursday, 26 March 2020, 23:56:48 GMT, Alex Lennon <notifications@github.com> wrote:  
 
 


I think we're also now revisiting a no-print foam idea as a basis for a splash protector mask. This could minimise machine time potentially and be more of an assembly production line task

**Comment by ajlennon** [comment]
I was put onto the guidance below. My CE guy has come back with his take on that guidance

>UK guidance published yesterday, https://www.gov.uk/guidance/regulatory-status-of-equipment-being-used-to-help-prevent-coronavirus-covid-19

    Face masks to protect the patient are Class 1 medical devices – not applicable
    Face masks to protect the wearer are PPE  - fair point, see below

 

Notes regarding providing a face shield:

    As discussed, the plan is to make something that is better than nothing
    Do NOT affix the CE mark
    Don’t make ANY claims regarding performance that you cannot substantiate with some test results or material datasheet for the plastic you are using if you trust your source has provided you with what they say they have
    Maybe describe it as “supplementary protection” or “to lower risk of direct contact” or something

**Comment by KarlDunkerley** [comment]
Rolls of polyprop just arrived. Will let you know how it works.

BTW Anyone know where to get rolls of elastic now Hobbycraft closed?

**Comment by amcewen** [comment]
@KarlDunkerley your links and photos sound interesting, but they don't seem to get pulled into the issue properly when they're emailed in(?).  You can see what it ends up like if you look at it on the web - e.g. https://github.com/DoESLiverpool/somebody-should/issues/1415#issuecomment-604950144

Don't know about elastic, but that sounds like a good thing to spin out as a new issue on the new repository - fill it out on https://github.com/DoESLiverpool/covid19/issues/new/choose and then anyone can go hunting and report back what they find without it getting lost in this long thread.

**Comment by KarlDunkerley** [comment]
Hi, Just started using it on the web. Had been having logging in issues.

220micron Poly feels about twice as thick as OHP, soft, flexible and a nice weight to it. Will try lasering after work finishes at 4pm. Have three large rolls if DoES wants a roll. Bugger to get cling film wrapper off.

**Comment by KarlDunkerley** [comment]
Would you be able to have a chat with this lady?

Related to the posting about the welsh group looking for supplies posted last night.

---------------------------------------------

Michaela Virgill

Gweinyddwr Gwithredol ● Operations Administrator

Hwb Gwyddorau Bywyd Cymru ● Life Sciences Hub Wales

3 Sgwâr y Cynulliad ● 3 Assembly Square

Bae Caerdydd, CF10 4PL ● Cardiff Bay, CF10 4PL

Rhif ffôn: 07503 643686 / 02920 467 030 ● Telphone Number: 07503 643686 / 02920 467 030

**Comment by RussCoty** [comment]
Not sure if I’ve missed anything here but is there a plan for hygiene in place during production? 

 Presumably that would have to be adressed  before they could be used.

**Comment by amcewen** [comment]
@RussCoty haven't worked anything out for that yet.  Want to spin up a [new issue for it](https://github.com/DoESLiverpool/covid19/issues/new) on the corvid19 repo?

**Comment by amcewen** [comment]
@KarlDunkerley I've pinged Liam, who's in N Wales and was the one asking last night to let him know.  Ta!

**Comment by KarlDunkerley** [comment]
Hi All,

Just done firm cuts using 220micron Polypropylene.

GOOD NEWS
Cut two visors with slits in 26 secs using my 500x300 50W Chinky Laser.
This was not optimised properly for cutting route.
Settings were 90mm/sec at 100% power.
DOES NOT smell of rotten fish when cut - just usual burnt acrylic pong. Acceptable.
Also, tried cutting with a craft knife and worked fine. 
Technically no need for laser so lots of people with a temple and cutting board could churn them out.

BAD NEWS
It throws off orange dust/strands that will need wiping off.
Couldn't cut two sheets at once at anything above 50% of the single sheet speed so no real time saving though no sign of welding together.
Had to use an MDF sacrifice board underneath to avoid flashback but barely scored it. Yippee.


COSTINGS
Used fully, the 10m x 1.25m roll would yield 34 x 6 = 228 visors for £30.00 making each one around 13-14p

Thoughts anyone?

**Comment by RussCoty** [comment]
Any thoughts on CNCing headbands? I think the laser cut ones will be faster but if there’s 3D  printing ones happening then we might as well look at CNC.

A simple profile with 4 nubs could work.

**Comment by ajlennon** [comment]
I wondered but don't know enough about CNC. It felt like it's a machine just waiting to be used if there was a way to do it

**Comment by KarlDunkerley** [comment]
You might be able to cnc the foam, especially if you go thicker than 10mm but anything else not worth it. Laser faster and cleaner at 3-6mm and sheets would end up everywhere.

FabLab Warrington at Beamont Collegiate Academy have a full size CNC if needed.

Managed today to get contacts for current managers of the lab if anyone needs them. Have kept in touch with the old manager.

Also have vinyl cutters, woodworkshop, milling and a laser but not a great one.

**Comment by KarlDunkerley** [comment]
MK II @42p each. Polypropylene, foam, elaccy band and a bit less velcro.

![Mk II 03](https://user-images.githubusercontent.com/34492815/77797263-b2638e00-7068-11ea-8310-b853a1ffb5ae.jpg)
![MK II 04](https://user-images.githubusercontent.com/34492815/77797265-b2fc2480-7068-11ea-851b-0eb51ffe91e6.jpg)

I think I've been spending too much time on this.....

**Comment by Dhiresh86** [comment]
What type of foam are you using? I wanted to add a tiny bit of foam to the 3d printed headbands

**Comment by ajlennon** [comment]
Looking very good @KarlDunkerley - how long is a sensible time for a person to make one of these if they were doing it all day ?

**Comment by KarlDunkerley** [comment]
Dhiresh86 It is EVA foam using the kind of interlocking gym mats you can find on eBay. About 3p each. Also called Plastizote. Laser cut for cleanness. Can be craft knifed but not always as clean. 10mm thick.

**Comment by KarlDunkerley** [comment]
> Looking very good @KarlDunkerley - how long is a sensible time for a person to make one of these if they were doing it all day ?

A skilled person, unlike me, cut craft knife this round a template in 20-30 seconds. Foam I would do on a laser in a batch but could also be cut manually in 10-20 secs. Velcro could be pre-cut into 20-25mm squares. Ditto elastic band. 

Most annoying thing was seperating velcro from backing. Would need to find a way to do this quick. My hands are all fingers and thumbs. 

If everything was set up properly and components prepared in batches, I would reckon around 90s secs to 120 secs.

With practice I think you could get to the 90secs mark, Around 30-40 per hour per person.

**Comment by KarlDunkerley** [comment]
I have been wondering if we should adopt a twin-track approach. Something like this prototype for the mass standardised use and the 3D or laser cutting for specialised versions eg needing a larger standoff from the face.

**Comment by KarlDunkerley** [comment]
BTW Couldn't get polyprop to fog up.

**Comment by magman2112** [comment]
I was talking to Karl earlier and I suspect we could likely reduce the making time by investing in a few jigs.

For example, we could design and laser cut a jig or template for cutting the clear visor. Similarly we could make a jig with multiple slots for cutting the foam strips from a single sheet.

So once a model is agreed we can then look to optimise the production.

**Comment by ajlennon** [comment]
In terms of time-lines @magman2112 is there anything we need to get in now that would facilitate the creation of these jigs? I'm assuming we don't need to know the specifics of the jigs to know the kinds of things we would make the jigs out of? Is that something we can kick off purchasing now?

**Comment by jackie1050** [comment]
Looking at some of examples in use where the screen is just an A4 sheet,  makers often just provide the frames and the hospital sorts out the rest.
________________________________

**Comment by magman2112** [comment]
It depends a lot in quantity required.

A basic jig could possibly be cut from the 3mm acrylic we have in stock, though I am not sure of the precise dimension of the visor Karl is currently using. This would likely work for 10’s to 100’s of visors with a quick fix of laser cutting more as required.

A more extreme jig would layer up and clamp say 10 to 20 sheets of visor material and use something like a router or plasterboard rotary saw to cut multiple visors at a time. A jig like this would produce multiple hundreds of. Visors a day.

**Comment by ajlennon** [comment]
>Looking at some of examples in use where the screen is just an A4 sheet,  makers often just provide the frames and the hospital sorts out the rest.

I _think_ this has the benefit it can be classed as self-build from discussions we've had, and thus we don't need to conform to all the CE requirements to provide a kit. But that would need confirming

**Comment by ajlennon** [comment]
> A jig like this would produce multiple hundreds of. Visors a day.

We appear to need 10,000's so I am wondering what we can do to leverage volunteers?

**Comment by amcewen** [comment]
Are you expecting us to supply the local area, nationally, globally?  If my twitter mentions is anything to go by, there are lots of other people running at this too.  Can we work out how we scale up production to 100s before we start planning to supply 10s of thousands?

It'd be good if I wasn't the only person deciding which of the designs work best, or what the pros and cons are of each one.

Once we're cranking out 100s of units, we'll know lots more about everything and can look to scale up then.

It'd also be good if you could all move over to the [new repository](https://github.com/DoESLiverpool/covid19) too, because it's really hard to keep up with the different conversations all mixed in one place here when you're only checking it a couple of times a day (like I am :-)

@karldunkerley nice to see you working out how to get photos in here, and getting better at your selfies too (aren't we all!) :grinning:

**Comment by ajlennon** [comment]
> Are you expecting us to supply the local area, nationally, globally? 

When I write "we" I mean society.

DoES isn't going to supply the demand.

An optimal process to go with should be a process that scales to supply the demand

**Comment by ajlennon** [comment]
> Are you expecting us to supply the local area, nationally, globally? 

When I write "we" I mean society.

DoES isn't going to supply the demand. Not even a tiny part.

An optimal process to go with should be a process that scales to supply the demand

**Comment by ajlennon** [comment]
>It'd be good if I wasn't the only person deciding which of the designs work best, or what the pros and cons are of each one.

Yeah we urgently need to clean this up into the repo to get feedback. I will try to help tomorrow

**Comment by ajlennon** [comment]
>It'd also be good if you could all move over to the new repository too

Can we close this thread then?

**Comment by KarlDunkerley** [comment]
My big concern at the mo is that any supplies of PETG or PP are likely to be in sheets 1500x1000 or rolls of 1.25m x 10m. Chopping these down to a manageable size for the worker bees to handle quickly and efficiently will be key.

We need a rig to feed these in, probably one by one, and then slice a strip to make visors, probably either approx 20 or 30cm depending on visor dimensions.

If someone could bolt together a rig for this out of either sheets of MDF or wood it would put my mind at rest.

Thoughts?

**Comment by KarlDunkerley** [comment]
![Some Heavy Shit](https://user-images.githubusercontent.com/34492815/77819705-c5ac4300-70d4-11ea-8af0-238250d3ef26.PNG)

**Comment by KarlDunkerley** [comment]
PETG on a pallet is some heavy shit. Half a tonne per pallet if this is correct.

Can anyone see anyone wrong with the calcs?

**Comment by KarlDunkerley** [comment]
This what Warrington FabLab are using their 3D printing for... https://www.facebook.com/fablabwarrington/photos/pcb.3206319782725812/3206319479392509/?type=3&theater

**Comment by KarlDunkerley** [comment]
Warrington Masks

![90677625_3206319486059175_6345757026710192128_n](https://user-images.githubusercontent.com/34492815/77820198-5d5f6080-70d8-11ea-8123-d3349bd7ac96.jpg)
![91006676_3206319559392501_395818112915603456_n](https://user-images.githubusercontent.com/34492815/77820199-5df7f700-70d8-11ea-8713-c0ebce83c5bf.jpg)

**Comment by KarlDunkerley** [comment]
Any chance Ade and Alex I could have your mbile nos? Mine is 07710 353 122. text me.

**Comment by ajlennon** [comment]
WhatsApp'd you @KarlDunkerley :)

**Comment by ajlennon** [comment]
@KarlDunkerley this is all great information - we are trying to break out of this massive thread to make the information more readily accessible on the repo. I will be working on this today. If  you can help with that it would be wonderful

**Comment by KarlDunkerley** [comment]
Saw the chaps at Warrington. They are only making the top valves for those masks. However, they do have their 8x4ft CNC up and running it we need to make large rigs for cutting rolls and sheets.

**Comment by KarlDunkerley** [comment]
Interesting thread on FB. Prusa on a small bed laser;

https://www.facebook.com/groups/888394261237059/permalink/2782287941847672/

**Comment by KarlDunkerley** [comment]
![Man in A Coke Bottle](https://user-images.githubusercontent.com/34492815/77844222-a7117f00-719c-11ea-93be-281fe536ebad.jpg)
Man in a Coke Bottle.

**Comment by KarlDunkerley** [comment]
This may be an odd thought but if we can't make these for the NHS, Can we make and sell them to the public in return for a donation to local health orgs and other volunteers?

**Comment by KarlDunkerley** [comment]
Adrian, Can you ask your fabric friend across the road if she has anything like this?

https://www.ebay.co.uk/itm/Manual-Leather-Cutting-Machine-Die-Cut-Leather-PVC-EVA-Sheet-Cutter-UK/223767600492?hash=item3419969d6c:g:Pg4AAOSwg~ReQlHE

**Comment by KarlDunkerley** [comment]
https://www.facebook.com/john.cabrer/videos/10157070720139212/

**Comment by seanmtracey** [comment]
Hey all, I'm just catching up with all of this now. Will work through the GH issues today and see if there's any input I can add from our effort down here.

Also, I've had an idea. Has anybody thought about constructing a UV light chamber to store these masks in? I don’t know what effect UV light has on viruses, but IIRC it can eradicate bacteria. Maybe it’s something we can put together to extend the shelf life of each mask?

**Comment by ajlennon** [comment]
Hey @KarlDunkerley - it would be great if you could put your thoughts on face masks into the issue we've created here - https://github.com/DoESLiverpool/covid19/issues/17

Hey @seanmtracey - great stuff thanks! we're trying to move away from this huge thread as it's ridiculously hard to parse. we've got a set of relevant issues here https://github.com/DoESLiverpool/covid19/ 

Please feel free to add onto them or if we're not capturing issues we need to add new ones

Gonna close this thread for now to try to move us over into the Covid19 space :)

**Comment by seanmtracey** [comment]
Sure thing! Will do.

**Comment by ajlennon** [comment]
(Edited that comment above to add on UV)
